roles/openshift_health_checker/library/aos_version.py

```diff
@@ -27,13 +28,18 @@ def bail(error):  # pylint: disable=missing-docstring
         module.fail_json(msg=error)
 
     rpm_prefix = module.params['prefix']
-
     if not rpm_prefix:
         bail("prefix must not be empty")
 
-    yb = yum.YumBase()  # pylint: disable=invalid-name
+    # determine what level of precision the user specified for the openshift version.
+    # should look like a version string with possibly many segments e.g. "3.4.1":
+    expected_version = module.params['version']
+    if not re.match(r'\A\d+(?:\.\d+)+\Z', expected_version):
```

Comment:
[`re.match`](https://docs.python.org/3/library/re.html#re.match) and `\A` in the pattern is redundant.